Makefile

```diff
@@ -416,6 +425,15 @@ coverage-html: coverage-unit-test
 coverage-view: coverage-unit-test
 	$(GO) tool cover -html=$(COVER_PROFILE)
 
+# coverage-view-patch generates an HTML file limited to changes and opens it in your browser.
+.PHONY: coverage-view-patch
+coverage-view-patch: GO_PKGS ?= $(shell ./scripts/go-packages-in-patch.sh)
+coverage-view-patch: $(SETUP_ENVTEST)
+coverage-view-patch: KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path --arch amd64 $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
```

Comment:
Aren't new people getting M1s? Is there anything here that detects arch?

Reply:
We might be able to get away with `GOARCH`, arch naming is pretty universal.

Reply:
Good suggestions both, I hadn't noticed that, it was a cut and paste error. I'll address it.